nova/src/prover.rs

```diff
@@ -30,16 +29,16 @@ impl<C: CircuitDriver> Prover<C> {
         r1cs: &R1cs<C>,
         relaxed_r1cs: &RelaxedR1cs<C>,
     ) -> (RelaxedR1csInstance<C>, RelaxedR1csWitness<C>, C::Affine) {
-        let mut transcript = Merlin::new(b"nova");
+        let mut transcript = MimcRO::<322, C::Base>::default();
```

Comment:
Where does this 322 come from?

Reply:
The 5.1 part describes how the number of rounds should be calculated. If we have an `n` bit field.
We use the Feistel mode with k equals to zero, so it's doubled.
https://eprint.iacr.org/2016/492.pdf#page=17

Reply:
Can you replace magic number 322 with constant variable with understandable naming?

Reply:
Done